src/rf/apps/workers/process.py

```diff
@@ -55,15 +51,20 @@ def start_polling(self):
                 if 'Records' in message and message['Records'][0]:
                     record = message['Records'][0]
                     job_type = record['eventName']
-                    if job_type == JOB_REPROJECT:
-                        delete_message = self._reproject(record['data'])
-                    elif job_type == JOB_HANDOFF:
+                    if job_type == JOB_HANDOFF:
                         # May want to keep the message from showing back up
                         # on the queue by setting a new visability with
                         # message.change_visibility()
-                        delete_message = self._emr_hand_off(record['data'])
+                        try:
+                            delete_message = self._emr_hand_off(record['data'])
+                        except KeyError:
+                            delete_message = False
```

Comment:
Is the try/catch for `record['data']` or for `delete_message = self._emr_hand_off(...)`? May want to split this up so we can distinguish between a message format error and an error during the EMR hand off job.